flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -228,7 +228,7 @@ MVN_COMMON_OPTIONS="-nsu -B -Dflink.forkCount=2 -Dflink.forkCountTestPackage=2 -
 e2e_modules=$(find flink-end-to-end-tests -mindepth 2 -maxdepth 5 -name 'pom.xml' -printf '%h\n' | sort -u | tr '\n' ',')
 e2e_modules="${e2e_modules},$(find flink-walkthroughs -mindepth 2 -maxdepth 2 -name 'pom.xml' -printf '%h\n' | sort -u | tr '\n' ',')"
 
-PROFILE="$PROFILE -Pe2e-travis1 -Pe2e-travis2 -Pe2e-travis3 -Pe2e-travis4 -Pe2e-travis5 -Pe2e-travis6"
+PROFILE="$PROFILE -Pe2e-travis1 -Pe2e-travis2 -Pe2e-travis3 -Pe2e-travis4 -Pe2e-travis5 -Pe2e-travis6 -Pe2e-hadoop -Pe2e-pre-commit"
```

Comment:
hmm I would probably put this into `build-apache-repo.yml`, to make it easier to add a hadoop-free nightly run. If you were to try that with this approach the test would naturally fail.

Reply:
Oops, I see. The nightly hadoop free build would have probably failed